docs/stdlib/array.rst

```diff
@@ -280,9 +279,7 @@ Reference
 
     :index: join array_to_string implode
 
-    Render an array to a string.
-
-    Join a string array into a single string using a specified *delimiter*:
+    Returns the elements of an ``array`` joined together with a ``delimiter``:
```

Comment:
Suggestion:
> Returns the elements of \`\`array\`\` as a string, joined together with a ``delimiter``.

- Removed "an" before \`\`array\`\` since we know the array being operated on.
- We should specify the result is a string.

Reply:
This suggestion will not work as it exceeds 79 chars on the first paragraph

Reply:
How about this?
> Returns ``array`` elements as a string joined by ``delimiter``